src/microbe_stage/PatchManager.cs

```diff
@@ -215,15 +214,6 @@ private void HandleCellSpawns(Patch patch)
         if (existing != null)
         {
             existing.Marked = true;
-
-            float oldFrequency = existing.Spawner.SpawnFrequency;
```

Comment:
What happened to this spawn frequency updating code?

Reply:
I admit I don't recall exactly since It's been a while since I made this change, but I do know that frequency and density were combined into one value, since they were just scalar multiples of each other anyway.

Reply:
Are they still correctly updated, though? This even has the logging for the spawn frequency change, so my worry is that your changes actually break the spawners from actually updating...

Reply:
It is not, which is probably what accounted for the overrepresentation Nunz noticed. I've pushed a fix to correct.

Reply:
Okay so we found a bug here. @Nunez2196 can you re-check that now?